web/src/main/java/com/bazaarvoice/emodb/web/resources/sor/DataStoreResource1.java

```diff
@@ -133,6 +137,32 @@ public boolean include(Table table) {
         };
     }
 
+    @GET
+    @Path ("_changedtables")
+    @Timed (name = "bv.emodb.sor.DataStoreResource1.listMetadataChangedTables", absolute = true)
+    @Unbuffered
+    @ApiOperation (value = "Returns all the emo tables whose metadata was changed.",
+            notes = "Returns a Iterator of a Map.",
+            response = Table.class
+    )
+    public Object listMetadataChangedTables(final @QueryParam ("from") DateTimeParam fromInclusiveParam,
+                                            final @QueryParam ("to") DateTimeParam toExclusiveParam,
+                                            final @QueryParam ("limit") @DefaultValue ("10") LongParam limitParam,
```

Comment:
I kind of alluded to this earlier, but what does "limit" mean?  Is it the maximum number of tables to return in this batch?  Or is it the maximum number of events?  For example, if there is one table that has 50 updates in a single day would the default return the one table with all 50 events or the first 10 events for the first table?